sbnanaobj/StandardRecord/classes_def.xml

```diff
@@ -67,7 +67,9 @@
    <version ClassVersion="10" checksum="83320005"/>
   </class>
 
-  <class name="caf::SRSlice" ClassVersion="21">
+  <class name="caf::SRSlice" ClassVersion="23">
+   <version ClassVersion="23" checksum="2124497341"/>
+   <version ClassVersion="22" checksum="4187618741"/>
```

Comment:
I believe you can just collapse these to ClassVersion 22 with the latest checksum no?